### PR TITLE
POL-179- 1965254 Dimension Information Missing for DAY and WEEK.

### PR DESCRIPTION
**Pull request by nshivappa**
### Description

This policy allows you to set up scheduled reports that will provide summaries of cloud cost across all resources in the billing centers you specify, delivered to any email addresses you specify. The policy will report the following:

### Issues Resolved

Fix for Bug Dimension Information Missing for DAY and WEEK.

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### Running template link and Screenshot:
Billing Term: Day
https://governance.rightscale.com/org/78/projects/30598/policy-incidents/5e259bf232401200016e0b80
image

Billing Term: Week
https://governance.rightscale.com/org/78/projects/30598/policy-incidents/5e259c097abe69000112c988
image

Billing Term: Month
https://governance.rightscale.com/org/78/projects/30598/policy-incidents/5e259bfe7abe69000112c987
image

### CODE REVIEW

**Comment by nshivappa** [comment]
@cdwilhelm,
Since there was a Danger ERROR for info block, I added info block in .pt file
info(
  version: "1.9",
  provider: "RS",
  service: "",
  policy_set: "Scheduled Report"
  )

File: cost/scheduled_reports/scheduled_report.pt [hunk 1]
```diff
@@ -14,10 +14,16 @@ _Note 1: The last 3 days of data in the current week or month will contain incom
...
 
 See [README](https://github.com/flexera/policy_templates/tree/master/cost/scheduled_reports) for more details"
-long_description "Version: 1.8"
+long_description ""
 severity "low"
 category "Cost"
 tenancy "single"
+info(
+  version: "1.9",
+  provider: "RS",
```
Comment (cdwilhelm) [inline cost/scheduled_reports/scheduled_report.pt]: Thanks for adding the info tag.  
Provider is "Flexera Optima"
There is no policy set, please remove the value.

File: cost/scheduled_reports/scheduled_report.pt [hunk 2]
```diff
@@ -14,10 +14,16 @@ _Note 1: The last 3 days of data in the current week or month will contain incom
 _Note 2: The account you apply the policy to is unimportant as Optima metrics are scoped to the Org._
 
 See [README](https://github.com/flexera/policy_templates/tree/master/cost/scheduled_reports) for more details"
-long_description "Version: 1.8"
+long_description ""
 severity "low"
 category "Cost"
 tenancy "single"
```
Comment (cdwilhelm) [inline cost/scheduled_reports/scheduled_report.pt]: please add default_frequency "monthly" and update the CHANGELOG that it's been added.

File: cost/scheduled_reports/scheduled_report.pt (hunk 1 above)
Comment (nshivappa) [reply cost/scheduled_reports/scheduled_report.pt]: Updated

File: cost/scheduled_reports/scheduled_report.pt (hunk 2 above)
Comment (nshivappa) [reply cost/scheduled_reports/scheduled_report.pt]: Added, Thanks

**Pull request merged**